packages/slice-machine/lib/builders/SliceBuilder/SideBar/index.tsx

```diff
@@ -91,14 +91,7 @@ const SideBar: React.FunctionComponent<SideBarProps> = ({
       >
         {isCheckingPreviewSetup ? <Spinner size={12} /> : "Open Slice Preview"}
       </Button>
-      {storybookUrl && (
-        <Link href={storybookUrl}>
-          <Button variant={"secondary"} sx={{ width: "100%", mt: 3 }}>
-            Open Storybook
-          </Button>
-        </Link>
-      )}
-      {!storybookUrl && !isPreviewAvailableForFramework && (
```

Comment:
Is the case where they don't have storybook and preview is not supported still covered?

Reply:
look like it

Reply:
The text will display and depending if the storybook is there or not we give a link to the documentation